src/components/views/auth/ModularServerConfig.js

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2019 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+'use strict';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import sdk from '../../../index';
+import { _t } from '../../../languageHandler';
+
+const MODULAR_URL = 'https://modular.im/?utm_source=riot-web&utm_medium=web&utm_campaign=riot-web-authentication';
+
+/**
+ * Configure the Modular server name.
+ *
+ * This is a variant of ServerConfig with only the HS field and different body
+ * text that is specific to the Modular case.
+ */
+module.exports = React.createClass({
```

Comment:
We should be using real classes for new code. Also I'm not sure if this Modular stuff should be in the react-sdk, but I'm also having trouble figuring out how it'd be separated out to riot-web :(

Reply:
Changed to regular classes as well. Good point about moving Modular out of the SDK... I filed https://github.com/vector-im/riot-web/issues/8297 for that.